src/lint.ts

```diff
@@ -246,26 +239,35 @@ export function hackLintMultidoc(inYaml: string, maybeOpts?: MultidocLintOpts):
   let offset = inYaml.indexOf(DOC_SEPARATOR) + DOC_SEPARATOR_LENGTH;
   const lineColumnFinder = lineColumn(inYaml);
 
-  return _.map(docs, (doc, index) => {
-    const linted = new Linter(doc, {
-      rules: opts.rules,
-      registry: opts.registry,
-      lineColumnFinder,
-      offset,
-    }).lint();
-
+  let index = 0;
+  for (const doc of docs) {
```

Comment:
If you need to keep track of the index, I would say just use a traditional for-loop

Reply:
yeah I guess. I don't know where my irrational (?) dislike for traditional for loops comes from.